README.md

```diff
@@ -43,6 +50,8 @@ Call via one of commands:
 ```shell
 cyclonedx-buildroot            # call script
 python -m cyclonedx_buildroot  # call python module CLI
+run((executable, '-m', 'cyclonedx_buildroot', "-i", "../tests/_data/manifest.csv", "-n", "My Project", "-v", "1.2.3.4", "-m", "company name"))
```

Comment:
why would you ad d this here? first, this is not executable on a sell,m and, this is internal!
it is in teh readme already: https://github.com/CycloneDX/cyclonedx-buildroot?tab=readme-ov-file#internals